server/server.go

```diff
@@ -79,12 +80,45 @@ var dirPaths = []string{
 	"trie",
 }
 
+// set log output to the designated file
```

Comment:
Comment should be in the form of:
`// setLogFileWriter sets the log output...`

Reply:
Fixed fe41b1e3945aa3ff7c35ff6095786ce8d717692c